src/components/PadSettings.spec.js

```diff
@@ -157,18 +157,20 @@ describe('PadSettings', () => {
       </MemoryRouter>
     );
     const padSelect = screen.getByRole('combobox', { name: 'select a pad' });
-    fireEvent.change(padSelect);
-    expect(padChange).toHaveBeenCalled();
     const colorSelect = screen.getByRole('combobox', {
       name: 'select a color',
     });
-    fireEvent.change(colorSelect);
-    expect(colorChange).toHaveBeenCalled();
     const sampleSelect = screen.getByRole('combobox', {
       name: 'select a sample',
     });
+    fireEvent.change(padSelect);
+    fireEvent.change(colorSelect);
     fireEvent.change(sampleSelect);
```

Comment:
One thing which came to my mind was why you use fireEvent instead of userEvent; it looks like the former is fine, but the ladder seems to be slightly better, [have a look here on firing events](https://testing-library.com/docs/dom-testing-library/api-events/) and on [userEvent](https://testing-library.com/docs/ecosystem-user-event/)

Reply:
I tried to access the functions with `userEvent.selectOptions()` but it didn't work.
With `fireEvent.change()` i get acces to the functions.
That's why I chose fireEvent.